spec/features/early_years_payment/practitioner/email_address_spec.rb

```diff
@@ -26,24 +26,6 @@
     expect(ActionMailer::Base.deliveries.count).to eq 2
   end
 
-  scenario "Resend passcode for verified email" do
```

Comment:
How come we're deleting this spec?

Reply:
i actually remove the page from the journey if the email has been verified, i don't see why one would verify the email again to an email they've already verified? if you hit the back link on next page it now takes them back to email page where they can change their email if they want